src/styles/block/is-active.scss

```diff
@@ -0,0 +1,19 @@
+.nav__item--is-active {
+  color: $hover-color;
+  position: relative;
+
+  &::after {
+    content: "";
+
+    position: absolute;
+    bottom: 0;
+    left: 0;
+
+    display: inline-block;
+    height: 4px;
+    width: 100%;
+
+    border-radius: 8px;
+    background-color: $hover-color;
+  }
+}
```

Comment:
it's not a separate BEM block, it's a modifier of element nav__item, move it to BEM block nav

Reply:
done